domain/model/list.go

```diff
@@ -0,0 +1,21 @@
+package model
+
+type ListDefinition struct {
+	List []List `json:"list"`
+}
+
+// Struct to unmarshal the Urban dictionary response of definitions based on a term
+// API For more infomation: https://rapidapi.com/community/api/urban-dictionary/
+type List struct {
```

Comment:
Where is this struct going to be used for? Since there is no CSV file to gather the info from